tests/bluetooth/controller/mock_ctrl/src/ull_central.c

```diff
@@ -42,7 +42,7 @@ int ull_central_reset(void)
 }
 
 uint16_t ull_central_iso_cis_offset_get(uint16_t cis_handle, uint32_t *cis_offset_min,
-					uint32_t *cis_offset_max)
+					uint32_t *cis_offset_max, uint16_t *conn_event_count)
```

Comment:
question only: this route is not used in unittesting, correct?

Reply:
It is used in ull_llcp_cc.c, so yes for cis unittests it is